lib/rules/declaration-block-no-ignored-properties/README.md

```diff
@@ -1,5 +1,7 @@
 # declaration-block-no-ignored-properties
 
+***Deprecated: this rule is outside the scope of stylelint's functionality. See [this comment](https://github.com/stylelint/stylelint/issues/2047#issuecomment-261382105) for details.***
```

Comment:
Let's point to http://stylelint.io/user-guide/release-planning/ here instead (and add something there about each of these deprecated rules)

This provides us with a single destination for `8.0.0` changes and planning, where we can create a consistent narrative that we can link to when needed.